skills_ml/algorithms/skill_extractors/section_extract.py

```diff
@@ -0,0 +1,43 @@
+from .base import SkillExtractor, CandidateSkill, CandidateSkillYielder
+from skills_ml.algorithms.string_cleaners.nlp import section_extract
```

Comment:
I've been thinking about to change the module name of `string_cleaners` because the functions there is not only for cleaning but vectorization or tokenization as well. Especially, now we have extraction. The name doesn't make sense anymore.

Reply:
That makes sense. I'm not sure what to change it to. Maybe nlp? Maybe we bring it up a couple levels? 
`skills_ml.nlp`?

I would split it up, but some of the items are just hard to categorize.